components/access-management/kfam/monitoring.go

```diff
@@ -0,0 +1,67 @@
+package kfam
```

Comment:
Do we expect other KF components to follow this pattern(same set of counter names, types etc)? If so, can it be put as a library outside `kfam`?

Reply:
We can put that effort into v1.0.1 and avoid blocking release of v1.0.0?
@jlewi

Reply:
+1 I suspect we will likely need to do some more refactoring post 1.0 as we will identify missing metrics etc... so I would be in favor of getting this merged now and then taking a more holistic look at our monitoring.